### ISSUE TITLE
Crash when Value contains special characters.

### ISSUE THREAD

**Issue by seiryokusan** [oxcafedead/barcode-reader-emulator#11]
I get this error if I use a special character anywhere in the Value.
```
java.lang.IllegalArgumentException: Invalid key code
	at java.desktop/sun.awt.windows.WRobotPeer.keyPress(Native Method)
	at java.desktop/java.awt.Robot.keyPress(Unknown Source)
	at oxcafedead.barcodereader.ui.HotkeyListeningThread.lambda$emulateBarCodeRead$2(HotkeyListeningThread.java:90)
	at java.base/java.lang.StringLatin1$CharsSpliterator.forEachRemaining(Unknown Source)
	at java.base/java.util.stream.IntPipeline$Head.forEachOrdered(Unknown Source)
	at oxcafedead.barcodereader.ui.HotkeyListeningThread.emulateBarCodeRead(HotkeyListeningThread.java:82)
	at oxcafedead.barcodereader.ui.HotkeyListeningThread.lambda$run$1(HotkeyListeningThread.java:65)
	at oxcafedead.barcodereader.keybind.windows.HotkeyBindManagerWindowsAdaptor.listen(HotkeyBindManagerWindowsAdaptor.java:50)
	at oxcafedead.barcodereader.ui.HotkeyListeningThread.run(HotkeyListeningThread.java:65)
```
Here is the exact setup then it occurred.
![image](https://user-images.githubusercontent.com/21677362/118707814-f83d6180-b7e8-11eb-9d03-32124757907a.png)

Thanks!

**Comment by oxcafedead** [comment]
Hi @seiryokusan.
Thanks for submitting this.
I've discovered few other issues nearby. For example, capitalized letters were not supported as well (the input was in lowercase).

I've tried to fix this in a version [0.4.1,](https://github.com/oxcafedead/barcode-reader-emulator/releases/tag/0.4.1) please try it.

I've also updated README, it now contains the full list of supported characters.

**Comment by seiryokusan** [comment]
It works!
Wow, what a quick fix. Thanks!
